StandardLibrary/runtimes/python/src/standard_library/internaldafny/extern/Time.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+import pytz
+import _dafny
+
+import standard_library.internaldafny.generated.Time
+import standard_library.internaldafny.generated.Wrappers as Wrappers
+
+class default__:
+    def CurrentRelativeTime():
+        return datetime.datetime.now(tz = pytz.UTC).timestamp() * 1000
+
+    def GetCurrentTimeStamp():
+        try:
+            d = datetime.datetime.now(tz = pytz.UTC).strftime("%Y-%m-%d'T'%H:%M:%S:%f'Z'")
```

Comment:
hmm this will create the timestamp at microsecond precision (good) but leaves the Z quoted, but Java and Dotnet don't. This may affect other operations like the hkerying

Reply:
Java does, maybe? https://github.com/aws/aws-cryptographic-material-providers-library/blob/main/StandardLibrary/runtimes/java/src/main/java/Time/__default.java
Let me look into this

Reply:
I removed the quotes and did some testing, I'm happy now

```
lucmcdon@80a997335f93 dafny % python3
Python 3.12.3 (main, May  9 2024, 13:24:59) [Clang 15.0.0 (clang-1500.3.9.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import datetime
>>> import pytz
>>> d = datetime.datetime.now(tz = pytz.UTC).strftime("%Y-%m-%dT%H:%M:%S:%fZ")
>>> datetime.datetime.fromisoformat(d)
datetime.datetime(2024, 7, 24, 16, 44, 3, 380474, tzinfo=datetime.timezone.utc)
>>> 

```